lib/gelf/index.js

```diff
@@ -13,10 +13,13 @@ function GELFMessageFactory (logObject, tag, source) {
   // GELF spec string (UTF-8) host - MUST be set by client library.
   const gelfRequiredFields = {
     'version': '1.1',
-    'host': source,
+    'host': logObject.source,
     'level': parseLevel(tag),
```

Comment:
What if logObject.source is undefined? Will the host appear in the graylog server or it will show as undefined?

Just a minor doubt, maybe a test case for this is at hand.

Reply:
the register function makes sure there's always a source by taking `os.hostname()` if `options.source` is either `undefined`, `null`, and the various other ways the ternary operator would switch to the else clause. calling `sendGelfMessage` on it's own is not possible. If somebody is using `GelfMessageFactory` on it's own, in a wrong way, then that's the users's fault and not how the plugin is supposed to be used.

Reply:
Agreed.